scripts/docs/vm-infra.md

```
@@ -0,0 +1,152 @@
+[dev-scripts](https://github.com/openshift-metal3/dev-scripts) and
+[metal3-dev-env](https://github.com/metal3-io/metal3-dev-env).
+
+## Network topology
+
+All subnets share the same N, taken from the third octet of `ENCLAVE_BMC_NETWORK`
+(e.g. N=5 from `100.64.5.0/24`). Subnet N is assigned atomically per cluster by
+`scripts/setup/allocate_subnet.sh`.
+
+### Connected mode — 2 bridges
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)